sample-app/src/components/DropdownSection.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from "classnames";
+import { useState, useEffect } from "react";
+
+export interface Option {
+  value: string,
+  display: JSX.Element
+}
+
+interface DropdownSectionProps<T> {
+  focusStatus?: string,
+  options: T[],
+  optionIdPrefix: string,
+  onFocusChange: (value: string, focusedOptionId: string) => void,
+  onLeaveSectionFocus?: (focusNext: boolean) => void,
```

Comment:
What do you think of `pastEnd` instead of `focusNext`? To me, `focusNext` implies that the focus should move to the next section, however there might not be another section (e.g. if it's the last section)

Reply:
I think that makes sense. I'll change it

Reply:
updated!